trace/trace.go

```diff
@@ -157,13 +165,15 @@ func NewSpanWithRemoteParent(name string, parent SpanContext, o StartOptions) *S
 
 func startSpanInternal(name string, hasParent bool, parent SpanContext, remoteParent bool, o StartOptions) *Span {
 	span := &Span{}
-	span.spanContext = parent
 
 	cfg := config.Load().(*Config)
 
 	if !hasParent {
```

Comment:
Since you added an else clause I think it would be clearer to invert this condition.

Reply:
Yeah I was thinking of that but didn't want to make a very colorful diff. I'll fix it. Thanks.